canonical-web-and-design/vanilla-framework
# Add tab content example

## Done

- Added a second example of the tabs pattern being used for panes of content, rather than navigation.

Fixes https://github.com/canonical-web-and-design/vanilla-squad/issues/886

## QA

- Open [demo](https://vanilla-framework-3733.demos.haus/docs/examples/patterns/tabs/content)
- See that the tabs show/hide panels as appropriate when clicked
- Review updated documentation:
  - [Component status](https://vanilla-framework-3733.demos.haus/docs/component-status)
  - [Tabs docs](https://vanilla-framework-3733.demos.haus/docs/patterns/tabs)

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
